jeanfrancoislarente/jf-test
# Initial success at PLAY! Summit on XM Cloud (#3)

* Adding CH connector files, database dats and transforms

* Updated to JSS 20.1.0-canary

* trying to turn off SSG to test

* change NextApiHandler to ApiHandler to fix build error

* adding debug flag for test output

* Switching SSG ENV check to be like other examples

* Updated package reference format to match what Alexander had in his test repo

* reverting back to no SSG at all

* testing - removing out own build yaml definition

* Resetting docker file back to where it started

* small change to trigger build

* Revert "testing - removing out own build yaml definition"

This reverts commit b46543e18864dc1c9773732ba54ee928892baefb.

* Removed shop

* Reverting one value for host

* Updating variables

* Adding fake file

* Adding health check for rendering

* Add healthz file

* return empty array from GetStaticPaths

* remove empty space

* Removing the jss app from config file since ENV varibles are not set

* Testing rending host override

* Needed to remove param to match ours

* Setting the app name in the magic rendering host replacement item

* Magic build param

* App name should be set by default by xm cloud build and set renderning host defaults for site node

Co-authored-by: Scott Mulligan <scm@sitecore.net>
Co-authored-by: Topaz Ahmed <mint@sitecore.net>
